exopy_hqc_legacy/instruments/drivers/visa/tektro_awg.py

```diff
@@ -451,6 +451,90 @@ def __init__(self, connection_info, caching_allowed=True,
                                   caching_permissions, auto_open)
         self.channels = {}
         self.lock = Lock()
+        self.AWG_FILE_FORMAT_HEAD = {
```

Comment:
Not sure what those are used for. In any case they could be simple class variables since they are global rather than instance variables.